src/configs/template.json

```diff
@@ -76,5 +74,8 @@
             }
         }
     },
+    "log_step": 500,
+    "verbosity": 2,
+    "saved_keys": ["data_input", "data_target", "model_output"],
```

Comment:
These keys are specifically for testing, which should be put in somewhere like data/mnist_test.json

Reply:
Fixed; train/test/eval configs are separated now